app/views/layouts/account_side_nav.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for(:nav) do %>
+  <%= render 'accounts/nav_auth', enable_mobile_nav: true, greeting: @view_model.header_personalization %>
+<% end %>
+
+<% content_for :content do %>
+  <div class='grid-row grid-gap mb3'>
+    <div class="desktop:grid-offset-3 desktop:grid-col-3">
```

Comment:
If this is intended to be centered, should the offset here be 2?

```suggestion
    <div class="desktop:grid-offset-2 desktop:grid-col-3">
```

( 12 - 5 - 3 ) / 2 = 2

Reply:
It was my best attempt at matching the designs which has it slightly off center: https://cm-jira.usa.gov/secure/attachment/123313/Two%20factor%20authentication.png